### DISCUSSION TITLE
Understanding Disk Utilisation on Search Node (Security Onion 2.3.90)

### DISCUSSION THREAD

**Discussion by karanandcobi** [Security-Onion-Solutions/securityonion#6482] [Unsupported Versions]
**Background:**
Hello Everybody, we are running a Distributed Grid (1 Manager, 1 Search and 1 Sensor Nodes) using VMware Workstation Pro in Air Gapped Environment. 
In our Search Node  **_log_size_limit_** value is set to 27xx GB in the minion pillar, because of this (As our ingestion rate is increasing and this limit has been reached) we can see **_so-curator-closed-delete_** in action leading to deletion of old closed indices which as a result keeps our NSM usage at 74~75% (looking at grafana and also df command).

**Question:** 
We have verified that the old indices are being deleting by looking at logs in **_/opt/so/log/curator/_**. However, the size of disk utilized at Virtual Machine level (Windows Drive) is continuously increasing. Can anybody explain this ?

Note: This drive is only used to store virtual drive of Search node.

Thank you in advance.

**Comment by InfosecGoon** [comment]
Sometimes disk issues arise in virtual environments if the guest operating system does not properly recognize file deletions occurring on the filesystem. If you compare the disk usage on /nsm using "df -h" and "du -sch /nsm" on the Search node, do the numbers match up?

--Matt
